cmd/wazero/wazero_test.go

```diff
@@ -251,6 +254,15 @@ func TestRun(t *testing.T) {
 <== (nread=5,errno=ESUCCESS)
 `,
 		},
+		{
```

Comment:
I believe we are missing the composed scope case `"--hostlogging=crypto,filesystem`